tracks/tracks-bot-dev.py

```diff
@@ -0,0 +1,48 @@
+from dotenv import load_dotenv
+import os
+import logging
+from telegram import Update
+from telegram.ext import Application, CommandHandler, MessageHandler, filters, ContextTypes
+
+load_dotenv()
+TOKEN = os.getenv("TELEGRAM_BOT_TOKEN")
```

Comment:
suggestion: we probably want to raise an error if the env variable is not set.

Note that this is a side-effecting line that will be run at the time of import. We are probably fine with that if we intend to start the application here, but better to avoid otherwise.